app/ante/cosmos.go

```diff
@@ -21,6 +23,9 @@ func newCosmosAnteHandler(options HandlerOptions) sdk.AnteHandler {
 			sdk.MsgTypeURL(&evmtypes.MsgEthereumTx{}),
 			sdk.MsgTypeURL(&sdkvesting.MsgCreateVestingAccount{}),
 		),
+		// democracy ante handler
```

Comment:
This is also optional, if you are allowing all messages in gov you can skip these 2 registrations